cellxgene_resources/curation_qa.ipynb

```diff
@@ -656,6 +656,133 @@
     "    ][['donor_id','development_stage_ontology_term_id']]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Check donor sex metadata against raw gene expression counts (genes found in PAR and NonPAR regions)**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Read in par regions\n",
+    "par_file = 'sex_analysis_regions/list_chrY_PAR_genes.txt'\n",
+    "nonpar_file =  'sex_analysis_regions/list_chrY_nonPAR_genes.txt'\n",
+    "par = pd.read_table(par_file, names=['par_genes'])\n",
+    "nonpar = pd.read_table(nonpar_file, names=['nonpar_genes'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Use raw matrix counts for sex analysis\n",
+    "raw_adata = ad.AnnData(adata.raw.X, var=adata.raw.var, obs=adata.obs)  # Create raw anndata if adata.raw exists otherwise raw_adata = adata\n",
+    "\n",
+    "#Extract regions from adata based on feature_name \n",
+    "par_adata = raw_adata[:,raw_adata.var.feature_name.isin(par['par_genes'])]  \n",
```

Comment:
This is going to be run on pre-submission Datasets sometimes, so we can't rely on 'feature_name' being present. So probably need to rely on the Ensembl IDs in the index because that will work for pre- and post-submission Datasets.